build/ci/phase-template.yml

```diff
@@ -4,13 +4,15 @@ parameters:
   buildScript: ''
   queue: {}
   customMatrixes: ''
+  defaultTestTargetFramework: netcoreapp2.1
```

Comment:
I don't think we should have this default here - it is duplicate of the default that is in the msbuild `.props` file: [https://github.com/dotnet/machinelearning/blob/master/test/Directory.Build.props#L5](https://github.com/dotnet/machinelearning/blob/master/test/Directory.Build.props#L5).

It would be great if the default is only put in a single place, so we only need to update a single place.